draft-ietf-sedate-datetime-extended.md

```diff
@@ -197,12 +197,13 @@ Timestamp:
 : An unambiguous representation of some instant in time.
 
 UTC Offset:
-: An amount of time before or after UTC that defines the relationship
-  between UTC and the local time in a particular place.
+: Difference between a given local time and UTC, usually given in
+  negative or positive hours and minutes. For example, local time in New
```

Comment:
Yep, that's exactly why "usually" is here, to allow for cases like that. But my understanding is that [RFC 3339 Section 4.2](https://datatracker.ietf.org/doc/html/rfc3339#section-4.2) specifically excludes representing sub-minute offsets in strings:

>       NOTE: Following ISO 8601, numeric offsets represent only time
>       zones that differ from UTC by an integral number of minutes.
>       However, many historical time zones differ from UTC by a non-
>       integral number of minutes.  To represent such historical time
>       stamps exactly, applications must convert them to a representable
>       time zone.

Honestly I find this text confusing because I'm not sure what "representable time zone" means; my assumption is that it means "pick the closest offset to the real offset"?  Also, I believe there was earlier discussion on the SEDATE group specifically excluded changing the string format to accommodate sub-minute offsets in the string.  If you want to re-open this discussion, you could but that's probably out of scope to this PR. 

That said, given that RFC 3339 already has a definition for what's called "Local Offset" in that spec, I'm now thinking that I should remove this definition here and just refer to RFC 3339's "Local Offset" definition instead.

Reply:
@cabo - in IETF specs, what's the usual process when a definition is re-used from another specification?  Is the definition added here too, with a note referring back to the definition in another RFC?  Or can the definition be omitted completely?

Reply:
At a minimum, you would need to specifically import the definition ("the terms ant, cat, and elk are defined in RFC4711").  If that definition is more than a click away (or, worse, buried in a paywalled document), I'd always fully cite it.

Reply:
> Also, I believe there was earlier discussion on the SEDATE group specifically excluded changing the string format to accommodate sub-minute offsets in the string.

That is infact the case. I haven't given up on it just yet, by the way, I'll still attempt to "backport" some of the updates from newer editions of ISO 8601 to IETF where I can once we have a working copy of the former with the liason relationship, but there's no point in blocking this draft for that potentially controversial set of updates (like sub-minute offsets here as well as extended years).

Reply:
I suspect the quoted ISO 8601 language means one of:
* turn your 1937-05-21 11:23+00:19:32.13's time and offset into 11:03:50.87Z or 12:03:50.87+01:00, or
* replace the offset with the zone that implies it, 1937-05-21 11:23[Europe/Amsterdam]

Referencing RFC 3339's definition overtly would clearly be a kindness to the reader, especially if you're opting to stick to its limitations. Since RFCs are fully open access, you don't need to worry about a paywall, but don't let that get in the way of quoting it.

Reply:
The definition of UTC offset is used in the following definitions only, so the question whether there should be UTC offsets with fractional minutes is somewhat orthogonal to this PR.  If you have a use case for fractional minutes, please do create an issue!